### PR TITLE
Backstop q4w exp

### PR DESCRIPTION
**Pull request by mootz12**
* Update SDK to pull in fix for emissions estimation
* add ledger entry restore functionality for q4w
* add tooltips and cleanup tooltip usage
* patch visual bug in positions table

### CODE REVIEW

**Comment by mootz12** [comment]
Deployed to testnet UI

**Comment by drew-boy** [comment]
APR tooltip doesn't need that comma, but it otherwise looks good

**Review by drew-boy — DISMISSED** [review]
neat

File: src/components/backstop/BackstopQueueItem.tsx [hunk 1]
```diff
@@ -57,19 +71,62 @@ export const BackstopQueueItem: React.FC<BackstopQueueItemProps> = ({
...
+    }
+  };
+
+  const queueItemActionButton = (sx: SxProps<Theme>) => {
+    const needsTooltip = !first || isRestore;
+    const tooltipMessage = !first
+      ? 'You can only unqueue the oldest withdrawal'
+      : 'This transaction ran into expired entries that need to be restored before proceeding.';
+
+    console.log('needsTooltip', needsTooltip);
```
Comment (Ryang-21) [inline src/components/backstop/BackstopQueueItem.tsx:87]: logs

**Review by Ryang-21 — CHANGES_REQUESTED** [review]

File: src/components/backstop/BackstopQueueItem.tsx (hunk 1 above)
Comment (drew-boy) [reply src/components/backstop/BackstopQueueItem.tsx:87]: change 'that' to 'which'

**Review by Ryang-21 — APPROVED** [review]

**Pull request merged**